Cargo.toml

```diff
@@ -30,7 +30,7 @@ cookie = { version = "0.14", features = ["percent-encode"], optional = true}
 once_cell = "1"
 url = "2"
 socks = { version = "0.3.2", optional = true }
-rustls = { version = "0.19", optional = true, features = [] }
+rustls = { version = "0.19", optional = true, features = ["dangerous_configuration"] }
```

Comment:
Rather than doing this by default, I think you can add another `rustls` import in the `dev-dependencies` that pulls in the extra feature.